sf-docker/.env

```diff
@@ -27,6 +27,6 @@ NoSqlDataStorageDisabled=false
 REGISTRY=registry.gitlab.com/innovatrics/smartface/
 
 # Version
-SF_VERSION=latest
-AC_VERSION=latest
-JETSON_VERSION=latest
+SF_VERSION=4.8.1
```

Comment:
These changes are caused by merging `release/4.8` into `dev`, which I hoped we wouldn't do, if only to keep `latest` docker versions in the `dev` branch.

But maybe this "benefit" is not worth the complexity of the workflow, seeing that many other changes (which should have gone to `dev` first) were also only merged to the `release/4.8` branch, for example the GPU on x64.

IMO we should talk about and really formalize the branching workflow, so we are all on the same page

Reply:
changed base to master_to_dev_conflict